api/apps/api/src/modules/scenarios/locks/scenario.lock.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity(`scenario_locks`)
+export class ScenarioLockEntity {
+  @PrimaryColumn({
+    type: `uuid`,
+    name: `user_id`,
+  })
+  userId!: string;
+
+  @PrimaryColumn({
+    type: `uuid`,
+    name: `scenario_id`,
+  })
+  scenarioId!: string;
+
+  @Column({
+    type: `date`,
+    name: `grab_date`,
```

Comment:
as this is brand new entity anyway, is there any reason to not change the column name as well? Am I missing anything?

Reply:
also `@CreateDateColumn({ name: 'created_at', type: 'timestamp without time zone' })`

Reply:
no, you're right, I meant to change the column name too but I forgot, will make the change :+1: